core-rust/mesh-api-server/test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Can we not commit this file for the sake of merging the PR soon?

Reply:
Done